### DISCUSSION TITLE
Final plot file format

### DISCUSSION THREAD

**Discussion by antcasq** [madMAx43v3r/mmx-node#42] [General]
I need to start creating my plots soon. I know if I do it now they should be usable in mainnet, however is there any plot format/optimization planned?

If format of the plots is going to change soon, for instance to use less disk space I might decide to postpone the creation of my plots. I will need 2 months to plot my drive, since I've a slow plotter ;)

**Comment by rhernaus** [comment]
The plot format is based on [chiapos](https://github.com/Chia-Network/chiapos) with a modification to allow unique plots per blockchain. Using a certain amount of space for a certain amount of proofs is what it is all about. It is not going to take less space in the future.
What is going to happen is that you will be able to create plots with plotnft to make poolable plots. But this will happen after launch of mainnet. So for now only OG solo plots.
